fixtures/index.js

```diff
@@ -32,7 +33,8 @@ window.aerogear = {
   agSync,
   agPush,
   gql,
-  ToggleNetworkStatus
+  ToggleNetworkStatus,
```

Comment:
I think we should physically put the device in airplane mode and not use a custom ToggleNetworkStatus

Reply:
Unfortunately for iOS devices in BrowserStack there is no way to put them offline, that is only possible for Android devices for now - https://www.browserstack.com/app-automate/network-simulation